src/components/Dashboard/TodoList.vue

```diff
@@ -1257,8 +1212,6 @@ export default class TodoList extends Vue {
   }
 
   private async loadConversion (task: ApiTaskIF): Promise<void> {
-    if (!this.isRoleStaff) return // regular users can't load a draft conversion
```

Comment:
Now show conversion item for reg users. This includes the conversion details but does not include any actions. (Otherwise the todo list is empty and it looks like users can file other things, which is incorrect.)

Example after fix:

![image](https://user-images.githubusercontent.com/10002889/207435830-e0cde10c-1596-4e6b-bec7-9c8c4f99b1f6.png)